### PR TITLE
[nan-467] add updatedAfter | updated_after param

### PR DESCRIPTION
**Pull request by khaliqgant**
## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: packages/shared/lib/utils/error.ts [hunk 1]
```diff
@@ -517,6 +517,11 @@ export class NangoError extends Error {
                 this.message = 'The sync is not currently running so cannot be cancelled';
                 break;
 
+            case 'incorrect_param':
+                this.status = 400;
+                this.message = `The parameter ${this.payload['incorrect']} is incorrect. Please ${this.payload['correct']}.`;
```
Comment (TBonnin) [inline packages/shared/lib/utils/error.ts]: what about `The param 'INCORRECT' is invalid. Do you mean 'CORRECT'?`

File: packages/shared/lib/services/notification/webhook.service.ts [hunk 2]
```diff
@@ -145,6 +149,7 @@ class WebhookService {
                 deleted: 0
             },
             syncType,
+            updatedAfter: dayjs(now).toDate().toISOString(),
```
Comment (TBonnin) [inline packages/shared/lib/services/notification/webhook.service.ts:152]: what does dayjs bring that cannot be done with Date?

**Review by TBonnin — APPROVED** [review]

File: packages/shared/lib/utils/error.ts (hunk 1 above)
Comment (khaliqgant) [reply packages/shared/lib/utils/error.ts]: Updated with https://github.com/NangoHQ/nango/pull/1777/commits/26d6d1e272416dd8e77f21a50fe95f0c2c782fc1

File: packages/shared/lib/services/notification/webhook.service.ts (hunk 2 above)
Comment (khaliqgant) [reply packages/shared/lib/services/notification/webhook.service.ts:152]: Consistency with how the records.service is handling it is all really.

**Pull request merged**